src/languages/es.js

```diff
@@ -476,10 +478,35 @@ export default {
             newPassword: 'Su contraseña debe tener al menos 8 caracteres, 1 letra mayúscula, 1 letra minúscula y 1 número.',
         },
     },
+    twoFactorAuth: {
+        headerTitle: 'Autenticación de 2 factores',
+        twoFactorAuthEnabled: 'Autenticación de dos factores habilitada',
+        whatIsTwoFactorAuth: 'La autenticación de dos factores (2FA) ayuda a mantener su cuenta segura. Al iniciar sesión, deberá ingresar un código generado por su aplicación de autenticación preferida.',
+        disableTwoFactorAuth: 'Deshabilitar la autenticación de dos factores',
+        disableTwoFactorAuthConfirmation: 'La autenticación de dos factores mantiene su cuenta más segura. ¿Estás seguro de que quieres desactivarlo?',
+        disabled: 'La autenticación de dos factores ahora está deshabilitada',
+        noAuthenticatorApp: 'Ya no necesitará una aplicación de autenticación para iniciar sesión en Expensify.',
+        stepCodes: 'Paso 1: Códigos de recuperación',
+        keepCodesSafe: '¡Mantenga estos códigos de recuperación seguros!',
+        codesLoseAccess: 'Si pierde el acceso a su aplicación de autenticación y no tiene estos códigos, perderá el acceso a su cuenta. \n\nNota: Configurar la autenticación de dos factores cerrará la sesión de todas las demás sesiones activas.',
+        stepVerify: 'Paso 2: Verificar',
+        scanCode: 'Escanea el código QR con tu',
+        authenticatorApp: 'aplicación de autenticación',
+        addKey: 'O agregue esta clave secreta a su aplicación de autenticación:',
+        enterCode: 'Luego ingrese el código de seis dígitos generado desde su aplicación de autenticación.',
+        stepSuccess: 'Paso 3: Terminado',
+        enabled: '¡La autenticación de dos factores ahora está habilitada!',
+        congrats: 'Felicidades, ahora tienes esa seguridad adicional.',
+    },
+    twoFactorAuthForm: {
+        error: {
+            pleaseFillTwoFactorAuth: 'Por favor, introduce tu código 2 factores',
```

Comment:
I think is missing a `de`, it should be:

`introduce tu código de 2 factores`

Reply:
Good one! The Expensify team is reviewing all Spanish translations in a separate document, so I will suggest this there